subprojects/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -114,13 +113,15 @@ public enum PathBase {
     private final DistributionLocator locator = new DistributionLocator();
     private final boolean isOffline;
     private boolean distributionUrlConfigured = false;
+    private final Property<Boolean> validateDistributionUrl = getProject().getObjects().property(Boolean.class);
```

Comment:
We don't need to explicitly create this property now that `Wrapper` is abstract.